modules/custom/improved_theme_settings/improved_theme_settings.module

```diff
@@ -252,7 +252,8 @@ function improved_theme_settings_get_setting($setting, $themename) {
   // Fetch setting from the theme.
   $themevar = Xss::filter(theme_get_setting($setting, $themename));
   // In this case '' is something different than 0.
-  if ($themevar !== trim((int) $themevar)) {
+  // @see: https://github.com/goalgorilla/open_social/pull/2748#discussion_r808805646
+  if (($themevar !== trim((string) (int) $themevar)) === is_numeric($themevar)) {
```

Comment:
I think my idea (in the linked comment) was to just simplify to the following:
```suggestion
  if (is_numeric($themevar)) {
```

Since the [truth table](https://dataschool.com/learn-sql/and-or-boolean-logic/#truth-tables) for that function is the same as for `$themevar !== trim((string) (int) $themevar)` I believe.

So this new if-statement I think is always TRUE, since the output is always equal.

Reply:
Done